multibindings.md

```diff
@@ -49,7 +50,6 @@ class Bar {
   @Inject Bar(Set<String> strings) {
```

Comment:
@gk5885 
I'm would hope that ```getABC()``` and ```"ABC"``` would be clear enough, but then I could also suggest the alternative of ```depA.getClass().getSimpleName();``` along with ```DepA.class.getSimpleName();``` in the assertion.

What do you think?